app/views/layouts/account_side_nav.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for(:nav) do %>
+  <%= render 'accounts/nav_auth', enable_mobile_nav: true, greeting: @view_model.header_personalization %>
+<% end %>
+
+<% content_for :content do %>
+  <div class='grid-row grid-gap mb3'>
+    <div class="desktop:grid-offset-3 desktop:grid-col-3">
+      <%= image_tag(
+        asset_path('user-access.svg'),
+        alt: '',
+      ) %>
+    </div>
+    <div class="desktop:grid-col-5">
+      <div>
+        Access your government benefits and services from your login.gov account.
+      </div>
+      <div>
+        <%= link_to 'Learn more about login.gov', MarketingSite.base_url %>
+      </div>
+    </div>
+  </div>
+  <div class='grid-row grid-gap'>
+    <aside class="desktop:grid-col-4">
+      <%= render 'accounts/side_nav' %>
+    </aside>
+    <div class='desktop:grid-col-8'>
```

Comment:
I've noticed mixed use throughout the codebase, but curious if it'd be valuable to try to settle on one or the other of single quotes or double quotes for HTML attributes.

```suggestion
    <aside class="desktop:grid-col-4">
      <%= render 'accounts/side_nav' %>
    </aside>
    <div class="desktop:grid-col-8">
```

Reply:
+1 I thin we should be consistent. I think it's probably a carryover from slim/rubocop styles (if those enforced single, except for interpolation or something like that)

Personally I'm used to double quotes in HTML

Reply:
My preference was double quotes as well for the same reasons, so I'll go with that.

Reply:
Thirded to double quotes 👍